modules/opencensus/pom.xml

```diff
@@ -116,4 +116,27 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-libs</id>
+                        <phase>test-compile</phase>
```

Comment:
Can you clarify the following questions, please:
- why phase `test-compile` instead of `package`?
- all the `opencensus` libs will be available at `libs` directory by default, right? should we copy them from e.g. `libs/opencensus` directory only when this integration is used?

Reply:
Hello @Mmuzaf ,

>why phase test-compile instead of package?

Changed to `package`. Thank you!

> all the opencensus libs will be available at libs directory by default, right?

All libs will be placed at `libs/optional/ignite-opencensus` in the release build.